### DISCUSSION TITLE
Include input in result of useAction hook

### DISCUSSION THREAD

**Discussion by TaylorFacen** [next-safe-action/next-safe-action#116] [Ideas]
Similar to how the `useAction` hook returns the `result` of the last run, it would be helpful for the hook to also return the input as well.

Let's say I have a command dialog of multiple items. When a user clicks on an item, I want to display a loading indicator on the specific item they selected. I could do this by using

```
const isLoading = status === 'executing' && input.itemId === itemId
```

**Comment by TheEdoRan** [comment]
Hey @TaylorFacen, I think that's a great suggestion actually, going to implement it soon.

**Comment by TheEdoRan** [comment]
Implemented in [v7.0.0-next.38](https://github.com/TheEdoRan/next-safe-action/releases/tag/v7.0.0-next.38)

**Reply by TaylorFacen** [reply]
Thank you!
